### DISCUSSION TITLE
How to change the sequence display "visualize" display table?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1486] [Q&A] [answered]
Hi @vincentbernat / expert,

Fist of all may i know what the default criteria sequence it take to display the table as per below? Is it "MAX" or "Average" or other thing. Second is it possible we change the sequence table display? For example we want "MAX" as on top sequence. Appreciate your feedback

![image](https://github.com/user-attachments/assets/d59766c1-613b-4322-937f-a353279e2c5e)

**Comment by vincentbernat** [comment, marked as answer]
It is sorted by average. It is not possible to sort by max.
